tools/cldr/cldr-to-icu/src/main/java/org/unicode/icu/tool/cldrtoicu/mapper/LocaleMapper.java

```diff
@@ -39,7 +40,7 @@
  * <p>This is currently driven by the {@code ldml2icu_locale.txt} configuration file via a
  * {@code RegexTransformer}, but could use any {@link PathValueTransformer} implementation.
  */
-public final class LocaleMapper {
+public final class LocaleMapper extends AbstractPathValueMapper {
```

Comment:
This class and SupplementalMapper were rejigged to use the new base class. This looks like a lot, but it's really only moving big blocks of code around, and making previously static methods instance methods.
Unfortunately the diff doesn't do a great job of spotting this (there's only about 10 lines actually different and everything else is code moving around a bit)